src/pages/about.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import About from "../components/About";
+
+function about() {
```

Comment:
Naming as above. 
Also, why is this file called about inside pages and there is an about inside components? What is the difference?